src/openrct2/interface/colour.c

```diff
@@ -66,14 +66,14 @@ void colours_init_maps()
 #ifndef NO_TTF
 static uint8 BlendColourMap[PALETTE_COUNT][PALETTE_COUNT] = {0};
 
-static uint8 findClosestIndex(uint8 red, uint8 green, uint8 blue)
+static uint8 findClosestPaletteIndex(uint8 red, uint8 green, uint8 blue)
 {
     sint16 closest = -1;
-    sint32 closestDistance = INT32_MAX;
+    double closestDistance = INT32_MAX;
```

Comment:
@marijnvdwerf Do you reckon we should drop the doubles in favour of going back to sint32s? They don't seem to have made much of an impact.

Reply:
I've just ran a test on all colours, int or double always produce the same outcome.